src/providers/VoiceFocusProvider/index.tsx

```diff
@@ -159,12 +159,18 @@ const VoiceFocusProvider: React.FC<Props> = ({
     }
 
     if (isVoiceFocusSupported) {
-      console.debug('Amazon Voice Focus is supported.');
+      console.info('Amazon Voice Focus is supported.');
     } else {
       console.warn('Amazon Voice Focus is not supported.');
     }
   }, [isVoiceFocusSupported]);
 
+  useEffect(() => {
+    if (voiceFocusDevice) {
+      console.info('Amazon Voice Focus transform device: ', voiceFocusDevice);
```

Comment:
Checking if this shows [Object object] in logs or how? Also, once you change the device does the earlier console log with earlier device printed still useful if someone provides us with the logs. If reference is updated then the earlier console log will also print the same device. Request you to check once.

Reply:
Yeah it will - you can print: JSON.stringify(voiceFocusDevice, null , 2);

Reply:
I have tested on meeting demo, it shows the device correctly. 
<img width="595" alt="Screen Shot 2021-10-04 at 3 14 44 PM" src="https://user-images.githubusercontent.com/82910472/135932134-83eb3b72-7101-4c91-901c-58ce2e1f7cb8.png">

Every time when VF transform device changes, it will print the current device. The earlier console log print the previous device, not current device. (Not sure if I understand your question correctly)